jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -73,8 +73,8 @@ def buildExclusions = [
   [ /sharedlibs_fips20/,              anyType,     gte(16) ],
 
   // OSX ---------------------------------------------------
-  [ /osx1015/,                        anyType,     gte(21) ],
-  [ /osx11-x64-release-tar/,          releaseType, lt(20)  ],
+  [ /osx11-x64-release-tar/,          releaseType, gte(18) ],
+  [ /osx11-x64-release-pkg/,          releaseType, gte(18) ],
```

Comment:
`gte(18)` will mean that these will be excluded (i.e. not run) on all our currently supported versions of Node.js, which isn't what we want. 

Highlighting again @targos 's comment in https://github.com/nodejs/build/pull/3531#discussion_r1367893562 
> In other words, if after this change all Node.js versions are supposed to be built on the same machine label, you can remove all osx-related lines from this file.

Reply:
ok, I see it now much clear :)

I removed the references to macOS in [ee12477](https://github.com/nodejs/build/pull/3531/commits/ee124777935dcdf229a6972f4e1afd8dd009b381) because we will only have macOS11 machines available and all the Node.js versions (18-21) should use them. So there is no need to restrict any machine